pulsar-broker/src/test/java/org/apache/pulsar/broker/service/SubscriptionSeekTest.java

```diff
@@ -493,4 +494,29 @@ public void testOnlyCloseActiveConsumerForSingleActiveConsumerDispatcherWhenSeek
         }
         assertTrue(hasConsumerNotDisconnected);
     }
+
+    @Test
+    public void testUpdateStatsForSingleActiveConsumerDispatcherWhenSeek() throws Exception {
```

Comment:
Is there anything related to seek? IMO, the test should move to org.apache.pulsar.broker.stats.ConsumerStatsTest

Reply:
@BewareMyPower I thought it's related to subscription consumer. You are right, it's the best to move to `org.apache.pulsar.broker.stats.ConsumerStatsTest`